docs/central-users.rst

```diff
@@ -210,12 +210,13 @@ From here, select :guilabel:`Retire User` and follow the on-screen instructions.
 Managing App Users
 ------------------
 
-App Users never gain any access to the management website: they do not have email addresses or passwords associated with their account, only a nickname so you can tell which is which. Once a Web User creates an App User within some project, a :doc:`configuration QR Code <collect-import-export>` will be generated which will grant a mobile device access to that project as that App User. Access can be revoked at any time, and Web Users can see which App Users uploaded which submissions.
+App Users never gain any access to the management website: they do not have email addresses or passwords associated with their account, only a Display Name so you can tell which is which.
 
-A newly created App User does not have access to any Forms. To give them access once they are created, use the :ref:`Form Access <central-projects-form-access>` tab on the Project. You will be able to allow access to particular Forms within the Project for each App User.
+Once a Web User creates an App User within some project, a settings QR code will be generated to grant a mobile device access to that project as that App User. The QR code includes a secret identifier just for that App User and should be kept private. See :doc:`settings QR code <collect-import-export>` documentation for more details.
+
+Access can be revoked at any time, and Web Users can see which App Users uploaded which submissions.
```

Comment:
`Web Users can see which App Users uploaded which submissions` seems very out of place and unrelated. I'd drop it since it's covered in the next section and move `Access can be revoked at any time` to the previous paragraph.